app/concepts/matestack/ui/core/collection/filter/submit/submit.haml

```diff
@@ -0,0 +1,3 @@
+%a{@tag_attributes, "@click": "submitFilter()"}
```

Comment:
Why not use a button here?

Reply:
because not everybody would like to use a button. using an `a` tag gives the matestack user more flexibility